.golangci.yml

```diff
@@ -1,11 +1,61 @@
+linters:
+  # please, do not use `enable-all`: it's deprecated and will be removed soon.
+  # inverted configuration with `enable-all` and `disable` is not scalable during updates of golangci-lint
+  disable-all: true
+  enable:
+    - asciicheck
+    - bodyclose
+    - deadcode
+    - depguard
+    # - dogsled (TODO: write rational for disabling here, brought over from Makefile)
+    # - dupl (TODO: write rational for disabling here, brought over from Makefile)
+    - errcheck
+    - exhaustive
+    - funlen
+    # - gochecknoglobals (TODO: write rational for disabling here, brought over from Makefile)
```

Comment:
We are currently using some globals. It's a lot more forgivable in a CLI tool than in a library. I hope most of them become obsolete with the current refactoring that I'm doing. Then we can enable this check again.